docs/src/framework/README.md

```diff
@@ -38,6 +38,9 @@ Most of these APIs fall into the following categories:
 To explore how these APIs come into play,
 see [the example of a 100-line kernel in safe Rust](a-100-line-kernel.md).
 
+The framework APIs have been extensively documented.
+You can view the complete API documentation at the [online documentation](https://asterinas.github.io/api-docs/aster_frame).
```

Comment:
I think we should have two API doc repos.

One is called `api-docs-nightly`, which, as its name suggests, reflects the most-up-to-date API docs. This is especially useful for us as Asterinas developers to read the most up-to-date docs. This repo is updated on every night. Updating on every change to `framework/` seems to be too high. To control the size of version history, we always reset the history as has been done by this PR.

The other is called `api-docs`. This repo keeps every published version of the API docs. This is probably more useful to users of Asterinas Framework as a user only cares about the API docs that she is currently working with. Apparently, uploading to this repo should only be done when the version number is bumped. The repo should be organized in the following form.

```rust
index.html
0.4.2/
0.4.3/
0.5.0/
```

The `index.html` file always redirects to HTML visit to the directory of the latest version.